regulations/docket.py

```diff
@@ -12,12 +14,12 @@
 
 def get_document_metadata(document_id):
     """ Retrieve document metadata from regulations.gov. """
-    # TODO Cache this as it hardly changes
     response = requests.get(
         settings.REGS_GOV_API_URL,
         params={'D': document_id},
         headers={'X-Api-Key': settings.REGS_GOV_API_KEY}
     )
+    logger.debug("Getting docket metadata")
```

Comment:
Very minor, but do you want to emit this log before or after the request is made?